include/RAJA/index/IndexSetBuilders.hpp

```diff
@@ -97,7 +98,8 @@ void buildLockFreeBlockIndexset(IndexSet& iset,
  *
  ******************************************************************************
  */
-void buildLockFreeColorIndexset(IndexSet& iset,
+void buildLockFreeColorIndexset( RAJA::TypedIndexSet<RAJA::RangeSegment,
```

Comment:
This one uses RangeSegment and List Segment only. Please remove stride segment type.

Reply:
That won't fix the bug, because LULESH is still using IndexSet.

Reply:
I know. We should change the type in lulesh. It should be easy because I think it is using a macro or typedef.